config/mainnetgenesis.go

```diff
@@ -1,5 +1,16 @@
+package config
```

Comment:
Why don't we keep this file under the mainnet directory? (same for testnet genesis)

Reply:
Not a bad idea... But not sure either 🤔 for me this is like the default node config, that also lives in this package